### ISSUE TITLE
Mapgen warning: "Map::getNodeMetadata(): Block not found"

### ISSUE THREAD

**Issue by Wuzzy2** [AiTechEye/xaenvironment#39]
In the console, this warning appears A LOT, usually, when a new part of the map generates.

I don't know exactly what this warning means, but if I remember correctly this can happen when the LuaVoxelManip tries to access an area of the map it was not supposed to access (ouf-of-bounds error). But I might be wrong.

Anyway, the fact this warning appears very often might be a sign that something deeper is wrong in the game.

ContentDB version: 2024-01-23

**Comment by AiTechEye** [comment]
yes this warnings is annoying, i tried to fix it for a while, i guess it comes from when the game tries to spawn buildings too far away, in some how

**Comment by Wuzzy2** [comment]
Ah yes, I think I remember. Minetest does not like it when you use a LuaVoxelManip to try to change an area outside of the area assigned to the LuaVoxelManip. All stuff done with LuaVoxelManip must be carefully reviewed to never ever go out of bounds.
